Documentation/Administration/Deployment/Tools/Deployer/_gitlab-deploy.yaml

```diff
@@ -1,4 +1,4 @@
-# Deployer Docs https://deployer.org/docs/7.x/basics
+# Deployer Docs https://deployer.org/docs/8.x/basics
```

Comment:
The file in the Gitlab Template points to 7.x as well: https://gitlab.com/gitlab-org/project-templates/typo3-distribution/-/blob/main/deploy.yaml?ref_type=heads